Longest_Increasing_subsequence_java/longest_increasing_subsequence_java.java

```diff
@@ -0,0 +1,48 @@
+package com.company;
+
+
```

Comment:
Please remove the package and the extra new-lines.
Also add documentation at the beginning of the file.